org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/general/completion/FilePathCompletionParticipant.java

```diff
@@ -259,4 +262,33 @@ private static void createFilePathCompletionItem(File f, Range replaceRange, ICo
 		response.addCompletionItem(item);
 	}
 
+	private static boolean hasPathBeginning(String currentText) {
+		if (currentText.startsWith("/")
+				|| currentText.startsWith("./")
+				|| currentText.startsWith("../")
+				|| currentText.startsWith("..\\")
+				|| currentText.startsWith(".\\")) {
+			return true;
+		}
+		return isAbsoluteWindowsPath(currentText);
+	}
+
+	private static boolean isAbsoluteWindowsPath(String currentText) {
+		if (currentText.length() < 3) {
+			return false;
+		}
+		char driveLetter = currentText.charAt(0);
+		if (driveLetter < 'A' || (driveLetter > 'Z' && driveLetter < 'a') || (driveLetter) > 'z') {
+			return false;
+		}

```

Comment:
Seems like `Character.isLetter(..)` should work here. In fact if you're going to try matching a letter, followed by ':' & '\\\\' , might it be easier to just use a regular expression, or just String#matches(..) ?

Reply:
Okay. I'm avoiding regexp according to Angelo's feedback. I'll switch to `Character.isLetter()`, that's way more legible that what I'm doing now

Reply:
Fair enough.